src/robotide/editor/settingeditors.py

```diff
@@ -190,7 +192,10 @@ def OnClear(self, event):
         self._controller.execute(ClearSetting())
         self._update_and_notify()
 
-    def update_value(self, event=None):
+    def _ps_on_update_value(self, message):
+        self.update_value()
+
+    def update_value(self):
```

Comment:
Why create a separate method instead of writing just message=None ?

Reply:
there're 2 reasons:

- the method arg `event` is never used in all project
- the overwritten method in class `DocumentationEditor` also does not have the arg, their args signature should be the same